plugins/modules/clustering/nomad/nomad_job_info.py

```diff
@@ -313,6 +313,7 @@ def run():
 
     changed = False
     nomad_jobs = list()
```

Comment:
```suggestion
```
I think we should get rid of this one, and directly add the elements to `result` in the loop below.

Reply:
Sure, I didn't realize it was a redundant assignment. It shoulf be fixed now.

Also I've added a changelog fragment.